x-pack/plugins/spaces/public/index.ts

```diff
@@ -6,16 +6,17 @@
 
 import { SpacesPlugin } from './plugin';
 
-export { Space } from '../common/model/space';
-
-export { GetSpaceResult } from '../common/model/types';
-
 export { SpaceAvatar, getSpaceColor, getSpaceImageUrl, getSpaceInitials } from './space_avatar';
 
 export { SpacesPluginSetup, SpacesPluginStart } from './plugin';
 
 export { SpacesManager } from './spaces_manager';
 
+export { GetAllSpacesOptions, GetAllSpacesPurpose, GetSpaceResult } from '../common';
+
+// re-export types from oss definition
+export type { Space } from '../../../../src/plugins/spaces_oss/common';
```

Comment:
Did you decide to keep the re-export in place? It looks like we're inconsistently importing from here and `spaces_oss` right now

Reply:
Yes since many other plugins already depend on Spaces X-Pack, it made sense to me to leave those dependencies and imports as-is. So just need to re-export the Space type from the Spaces X-Pack plugin. The saved objects tagging plugins behave this way as well and thought it would be simplest to emulate that behavior.